components/NavPagination/NavPagination.js

```diff
@@ -0,0 +1,7 @@
+export default function NavPagination(page, maxPage) {
+  const pagination = document.createElement("span");
+  pagination.classList.add("navigation__pagination");
+
+  pagination.textContent = `${page} / ${maxPage}`;
```

Comment:
You can also hardcode here '1 / 1' so you don't need the parameter. This textContent will be overwritten with the first fetch.